src/or/command.c

```diff
@@ -534,6 +607,13 @@ command_process_relay_cell(cell_t *cell, channel_t *chan)
     log_fn(LOG_PROTOCOL_WARN,LD_PROTOCOL,"circuit_receive_relay_cell "
            "(%s) failed. Closing.",
            direction==CELL_DIRECTION_OUT?"forward":"backward");
+
+    /* This was missing from the original cell code. Why? */
```

Comment:
Because `circuit_receive_relay_cell` was just called above, and that function would handle any additional places where the cell event should get emitted. We don't want to emit multiple cell events for the same cell.